src/app/ApplicationTemplate.Mobile/ApplicationTemplate.Mobile.csproj

```diff
@@ -201,6 +211,14 @@
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon.png" />
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%402x.png" />
 				<BundleResource Include="iOS/Resources/Assets/splashscreen_icon%403x.png" />
+				<BundleResource Include="iOS\Resources\Fonts\Guillon-Black.ttf" />
```

Comment:
I'm curious the slash are not the same directions for the Assets and the Fonts, does that change something?

Reply:
That's my bad, both are working, but it's a mix of VS adding the path by itself and me copying from other configuration file.

Reply:
Will be cleaning this later today.